proto/executor.proto

```diff
@@ -61,6 +61,13 @@ enum ExchangeType {
 	Hash = 2;
 }
 
+// Data compression mode
+enum CompressionMode {
+	NONE = 0; // no compression
+	FAST = 1; // fast compression/decompression speed, compression ratio is lower than HC mode
+	HIGH_COMPRESSION = 2; // high compression (HC) ratio mode
```

Comment:
Maybe we should use the name of the compression algorithm as the enum value, which is more easier if we wants to add more compression algorithms.

Reply:
Originally, I use compression method name directly. But the real codec format is implemented by TiFlash itself and other components can't recognize. If we need to change compression method for one mode, we can just update [MPPDataPacket::version](https://github.com/pingcap/kvproto/blob/7cd28226c2a21e489b077a87e1f5c9ba2c950944/proto/mpp.proto#L76) and use new format.

Reply:
Compression method like `snappy`, `LZ4`, `LZ4-xxx` are in same family. I think we can use mode `FAST` to present it. We can let user choose one mode by tidb session and let tiflash decide the actual method.

Reply:
OK